redisinsight/ui/src/components/monitor/Monitor/Monitor.tsx

```diff
@@ -10,12 +11,15 @@ import {
 import { AutoSizer } from 'react-virtualized'
 
 import { IMonitorDataPayload } from 'uiSrc/slices/interfaces'
+import { ReactComponent as NoPermissionsIcon } from 'uiSrc/assets/img/monitor/no_permissions.svg'
```

Comment:
It might be better to name this icon `ban,svg`. This is a very common name https://fontawesome.com/icons/ban?s=regular